### PR TITLE
Os/feature votes list and pages

### PR DESCRIPTION
**Pull request by saienkoo**

### CODE REVIEW

File: resources/assets/js/app/controllers/IdeaHubController.js [hunk 1]
```diff
@@ -1,9 +1,67 @@
...
+var Radio = require('backbone.radio');
 
-module.exports = Marionette.Object.extend({
+var currentUser = require('../initializers/currentUser');
+
+var VoteModel = require('../models/VoteModel');
+var CommentModel = require('../models/CommentModel');
+
+var CommentsCollection = require('../collections/commentCollection');
+var VoteAICollection = require('../collections/VoteAICollection');
```
Comment (binaryvo) [inline resources/assets/js/app/controllers/IdeaHubController.js]: Why 2 files with collections starts with different register?

**Comment by saienkoo** [comment]
Fixed

**Comment by binaryvo** [comment]
Please up to day your branch with next

File: resources/assets/js/app/controllers/IdeaHubController.js [hunk 2]
```diff
@@ -1,9 +1,67 @@
...
+
+var ListVotes = require('../views/votes/ListVotes');
+var ShowVote = require('../views/votes/ShowVote');
+
+var Votes = require('../instances/Votes');
 
+module.exports = Marionette.Object.extend({
+    initialize: function () {
+        this.listenTo(Radio.channel('votesChannel'), 'createComment', function (view) {
+            var model = new CommentModel({user_id: currentUser.get('id'), rating: 0}, {parentUrl: view.options.collection.parentUrl});
```
Comment (binaryvo) [inline resources/assets/js/app/controllers/IdeaHubController.js:21]: You can move property rating:0 to model property 'defaults'

**Pull request closed without merging**